addons/weapons/script_component.hpp

```diff
@@ -17,4 +17,4 @@
 #define ATT_POINTER_OPTRE_RAIL_M12 "OPTRE_M12_Laser", "OPTRE_M12_Flashlight", "OPTRE_M12_Vis_Red_Laser"
 #define ATT_POINTER_OPTRE_RAIL "OPTRE_BMR_Laser", "OPTRE_BMR_Vis_Red_Laser", "OPTRE_BMR_MEQ_Flashlight"
 #define ATT_POINTER_OPTRE_MUZZLE "OPTRE_BMR_Flashlight", "OPTRE_BMR_Flashlight_Laser", "OPTRE_BMR_Flashlight_Vis_Red_Laser"
-#define ATT_POINTER_ACE "acc_pointer_IR", "ACE_acc_pointer_red", "ACE_acc_pointer_green_IR", "ACE_acc_pointer_green"
+#define ATT_POINTER_ACE "acc_pointer_IR", "ACE_acc_pointer_red", "ACE_acc_pointer_green_IR", "ACE_acc_pointer_green", "ACE_DBAL_A3_Red", "ACE_DBAL_A3_Green", "ACE_SPIR"
```

Comment:
Go ahead and remove all of these `ATT` macros except for ATT_POINTER_OPTRE_RAIL unless you want to migrate the ma5 to use the class format of attachments as well.